ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -172,6 +228,21 @@ class AlertsView extends React.Component {
     let updates = { loading: false };
     const params = this.composeParams(id, page, framework, status);
 
+    if (this.isListMode()) {
+      if (filterText !== '') {
+        params.filter_text = filterText;
+      }
+      if (hideImprovements) {
+        params.hide_improvements = hideImprovements;
+      }
+      if (hideDownstream) {
+        // params.hide_downstream = hideDownstream;
```

Comment:
I think that the `// params.hide_downstream` comment is something which got forgotten as a comment.